### ISSUE TITLE
Create a CLI 

### ISSUE THREAD

**Issue by joshcarp** [joshcarp/llm.go#3]
- A user-friendly interactive command-line interface, similar to ollama
- Use cobra or charm.sh

Flags:
To start with it would only need a `--prompt` flag, then interactive chat style would be a further goal

After the simple `--prompt` flag, it would be nice to be able to extract information like token loss, perplexity, and any other information that can be extracted
- `--temperature`
- `--batch-size` controls size of batch size, larger would lead to quicker passes
- `--training-data` would specify training data to open
- `--eval-data` would specify data to eval against

See https://github.com/joshcarp/grpctl for reference

**Comment by P4nos** [comment]
Hi, this project looks cool and I would like to attempt to fix this issue, if that's alright.

Just to clarify, this `prompt` flag is meant to be used as a mechanism that passes user input to the model for inference. Did I get this right?

Thanks.
